js/reports.jsx

```diff
@@ -93,7 +96,7 @@ var HistoryReport = React.createClass({
   },
 
   getData: function(id) {
-    var url = "http://127.0.0.1:8000/api/v1/resultview/?format=json&limit=0&runcaseversion__run__series=" + id
+    var url = "http://10.247.24.126:8181/api/v1/resultview/?format=json&limit=0&runcaseversion__run__series=" + id
```

Comment:
Although this is my bad, but could you use `js/configs` instead? should be something like `config.baseUrl`. Thanks.